config/kubefedconfig.yaml

```diff
@@ -5,9 +5,13 @@ metadata:
   namespace: kube-federation-system
 spec:
   scope: Cluster
+  controllerDuration:
```

Comment:
Would it be possible to avoid duplicating these defaults here and have them instead be set by the controller manager when loading config from a file?

Reply:
There was at least one user that was wanting to run the controller manager outside of the cluster for some reason I can't recall, which suggested that providing the config via argument wasn't strictly test-only.  But given that we're already supporting loading `--kubefed-config`, adding a call to set defaults after unmarshaling seems cheaper than to me than having to maintain defaults manually.